source/sass/blocks/logo.scss

```
@@ -13,3 +13,11 @@
 .logo__text {
   justify-self: center;
 }
+
+.logo[href]:hover {
+  opacity: 0.8;
+}
+
+.logo[href]:active {
+  opacity: 0.6;
+}
```

Можно еще так
```suggestion

.logo:link:hover {
  opacity: 0.8;
}

.logo:link:active {
  opacity: 0.6;
}
```

Формально это получается как бы правильнее, хотя работать будет и то и то